packages/testing/src/mocks/mockedApi.ts

```diff
@@ -375,13 +379,18 @@ export function getMockedApi(): MockApiPromise {
           )
         ),
       },
+      publicCredentials: {
+        credentials: jest.fn(),
+        credentialSubjects: jest.fn(),
+      },
     },
     runtimeMetadata: {
       asV11: {
         modules: [],
       },
     },
     registry: TYPE_REGISTRY,
+    createType: TYPE_REGISTRY.createType,
```

Comment:
I just removed it since we don't need it anymore anyway: https://github.com/KILTprotocol/sdk-js/pull/677/commits/9ebc427e6823cf30c0ea59ee4e8d6b59b1f50d7e